testsuite/helloworld/tests/sandbox_c_format_open/format_open.c

```diff
@@ -0,0 +1,17 @@
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+int main() {
+    int fd;
+    if ((fd = open("%s", O_RDONLY)) >= 0) {
+        close(fd);
+        puts("Should not open the file");
+    } else if (errno == EACCES || errno == ENOENT) { // differs between seccomp and landlock
+        puts("Hello, World!");
+    } else {
+        printf("Wrong errno: %d", errno);
+    }
```

Comment:
Missing return. These may be just tests, but still...

Reply:
I've added return codes based on what happened.